stdlib/ballerina-builtin/src/main/ballerina/ballerina/io/natives.bal

```diff
@@ -64,6 +70,19 @@ public struct SocketProperties {
     string sslProtocol;
 }
 
+@Description {value:"Opens a byte channel from a specified file location."}
+@Param {value:"path: path to the file location."}
+@Param {value:"accessMode: whether the file should be opened for read,write or append."}
```

Comment:
Shall we define an Enum for the access mode? Having it as a string is difficult to remember to the user and allow no compile time checks

Reply:
Yes we discussed on this, we will be adding an enum with the effort which will be put to unify the file apis with io.